### PR TITLE
Fix Issues with log-configs

### PR DESCRIPTION
**Pull request by dulanjalidilmi**
## Purpose
Fix https://github.com/wso2/micro-integrator/issues/3351

Screenshot 2024-06-12 at 10 49 28 AM

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/core/rest/model/LogConfigDetail.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.ei.dashboard.core.rest.model;
+
+import javax.validation.Valid;
+
+
+import io.swagger.annotations.*;
```
Comment (arunans23) [inline monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/core/rest/model/LogConfigDetail.java:24]: Shall we list down individual imports?

File: monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/core/rest/model/LogConfigDetail.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+import java.util.Objects;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+public class LogConfigDetail   {
```
Comment (arunans23) [inline monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/core/rest/model/LogConfigDetail.java:29]: Shall we reformat this class? The formatting seems to be a little off in some places?

File: monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/core/rest/model/LogConfigDetail.java (hunk 1 above)
Comment (dulanjalidilmi) [reply monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/core/rest/model/LogConfigDetail.java:24]: Fixed.

File: monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/core/rest/model/LogConfigDetail.java (hunk 2 above)
Comment (dulanjalidilmi) [reply monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/core/rest/model/LogConfigDetail.java]: This is fixed.

**Pull request merged**